### PR TITLE
Update README File To Include Config Instructions

### PR DESCRIPTION
**Pull request by amayer5125**
Remove outdated links. Replace with config directions.

### CODE REVIEW

**Comment by PlasmaPower** [comment]
You might want to add something about how to make it run automatically (probably cron for linux and mac, and the built in program Task Scheduler for Windows).

**Comment by amayer5125** [comment]
@sealemar Here is what I have so far. This needs some work. Im not sure the best way to document events. And I have no idea how you run this on windows.
I need some suggestions on other things to add.

**Comment by PlasmaPower** [comment]
Also, you might want to check your git config for your email, because the commit doesn't link to your user.

**Comment by amayer5125** [comment]
@PlasmaPower do you have a good example of cron job? I remember there was some argument about how to do it previously.
I used my old email address. I will update it in my config on my server. Thanks. Just curiouse tho where are you trying to link from?

**Comment by PlasmaPower** [comment]
If you click on [the commit](https://github.com/amayer5125/BingRewards/commit/702c78ac87ada251e6dfb205c322fa93d4576eb7), the author just is your name, and is not your Github account. If you look at [another commit](https://github.com/sealemar/BingRewards/commit/f46e2b84a61d8463be66d0141065a2149bc590e2), you can see it links back to the Github profile.

**Comment by PlasmaPower** [comment]
A note about python 2 being required (and in your example in the path) would be nice.

**Comment by PlasmaPower** [comment]
`* */3 * * * [command goes here]` should execute the command every 3 hours.

**Comment by amayer5125** [comment]
@PlasmaPower I know thats how you run a cron job every 3 hours. wasnt there something about running after random sleep and including a path?

**Comment by PlasmaPower** [comment]
From [a link] on the old readme:
```
PATH=/usr/bin:$PATH
LOCAL_CONFIG_DIR=/home/bingrewards/etc

0   1   *   *   *   sleep $(($RANDOM \% 120))m && python2 /home/bingrewards/bin/main.py 2>&1 | gzip > /home/bingrewards/var/log/bingrewards/`date "+\%Y-\%m-\%dT\%H:\%M:\%S"`.log.gz
```

**Comment by amayer5125** [comment]
@PlasmaPower I added the cron job. I also added a note about python 2

**Comment by sealemar** [comment]
@amayer5125 you may also want to add

```
SHELL=/bin/bash
```

as the first line to the ```cron``` from [this post](https://github.com/sealemar/BingRewards/pull/123#issuecomment-78083890) by @PlasmaPower

**Comment by amayer5125** [comment]
@sealemar does this cron look better?

File: README.md [hunk 1]
```diff
@@ -1,20 +1,57 @@
-BingRewards
-===========
-
-Automation to earn Bing reward points with Facebook and Live credentials.
-
-- For more information, including how to use this, please, take a look at my blog post:
```
Comment (sealemar) [inline README.md]: please, don't remove the links. Let's have both the links and your portion of README

Comment (sealemar) [inline README.md]: You can say that to have a detailed (but may be partially outdated) information - read the links. The rest of the README will focus on a brief introduction on how to have the script installed and running in a few minutes.

**Comment by amayer5125** [comment]
@sealemar I believe I got all the changes you listed. Is there anything else?

**Comment by sealemar** [comment]
@amayer5125 good job on the ```README```, please consider a few more changes

**Comment by amayer5125** [comment]
@sealemar I made the requested changes. Is there anything else?

**Pull request merged**